lib/active_merchant/billing/gateways/reach.rb

```diff
@@ -56,6 +56,18 @@ def capture(money, authorization, options = {})
         commit('capture', post)
       end
 
+      def supports_scrubbing?
+        true
+      end
+
+      def scrub(transcript)
+        transcript.
+          gsub(%r(((MerchantId%22%3A%22)[\w-]+)), '\2[FILTERED]').
+          gsub(%r((signature=)[\w%]+), '\1[FILTERED]\2').
+          gsub(%r((Number%22%3A%22)[\d]+), '\1[FILTERED]\2').
+          gsub(%r((VerificationCode%22%3A)[\d]+), '\1[FILTERED]\2')
```

Comment:
All of these look good, but we might need to consider other payment method types and if the transcript for those will have additional data that we need to scrub.

Reply:
@naashton for Reach the initial implementation is only going to consider CC, if new payment methods ara added in the future, the natural thing to do is to update the scrub method.

Reply:
Yes, I think you are correct that we need to handle these after we implement alternate payment methods. Just wanted to make sure I called it out, mostly so that I don't forget 😅